packages/kbn-legacy-logging/src/get_logging_config.ts

```diff
@@ -27,11 +27,14 @@ export function getLoggingConfiguration(config: LegacyLoggingConfig, opsInterval
     });
   } else if (config.verbose) {
     _.defaults(events, {
+      error: '*',
       log: '*',
       ops: '*',
-      request: '*',
-      response: '*',
-      error: '*',
+      // To avoid duplicate logs, we explicitly disable these in verbose
+      // mode as they are already provided by the new logging config under
+      // the `http.server.Kibana.response` context.
+      request: '!',
+      response: '!',
```

Comment:
We [had discussed doing this](https://github.com/elastic/kibana/issues/13241#issuecomment-757330953) as a means of preventing duplicate log entries where possible, however I just wanted to call it out before merging as _technically_ it's a breaking change since the new log format is slightly different from the old one.

Folks using `logging.verbose` will still get response logs, they'll just look a bit different.

Reply:
> it is not necessarily so

Yea, but that would require manual configuration change from the user, so unsure this is good enough to not consider that breaking?

Reply:
> that would require manual configuration change from the user, so unsure this is good enough to not consider that breaking?

Yeah this is the question I was getting at. Totally agree there are workarounds to minimize disruption (custom `pattern`, or manually setting `logging.events.response`) - the question is whether we are okay forcing users to make a manual configuration change to restore the old format (even if the message contents are essentially the same).

Or I guess what we are really asking: Is it worse to inconvenience users with duplicate log messages (which are already quite chatty), or to break a log message format (which they could manually fix).

Reply:
> Yea, but that would require manual configuration change from the user

we can leverage logging service API to customize the pattern. WDYT ?
https://github.com/elastic/kibana/blob/2b5071ff36da10af6b9bd40864700d36d30e51ec/x-pack/plugins/security/server/audit/audit_service.ts#L222-L240

Reply:
@restrry Just to be clear, you mean apply a custom pattern to the http services response logs to format them in a way that's BWC with legacy logs, but only when using verbose logs & stdout? 

I guess the tradeoff would then be that users relying on the _new_ http response logs would suddenly have a different message format when verbose=true, which wouldn't technically be breaking as it is a new set of logs, but might be confusing nonetheless.

FWIW, I don't think introducing duplication would be the end of the world if we needed to go that route... I really could go either way on this.